**EXPLORATORY ANALYSIS ON AIRBNB BAROSSA VALLEY DATASET**

**<span lang="EN-AU" style="font-size: 10.5pt; font-family: &quot;Calibri(Body)&quot;, serif; color: black;">What to expect from this project?</span>**

<span lang="EN-AU" style="font-size: 10.5pt; font-family: &quot;Calibri(Body)&quot;, serif; color: black;">-Data Cleaning and Pre-processing</span>

<span lang="EN-AU" style="font-size: 10.5pt; font-family: &quot;Calibri(Body)&quot;, serif; color: black;">-Statistical Analysis</span>

<span lang="EN-AU" style="font-size: 10.5pt; font-family: &quot;Calibri(Body)&quot;, serif; color: black;">-Exploratory Analysis</span>

**<span lang="EN-AU" style="font-size:
10.5pt;font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">Context</span>**

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">In this project, we aim to use SQL to clean and analyze the Airbnb Barossa Valley dataset, a popular tourist destination known for its picturesque landscapes, renowned wineries, and rich cultural heritage. We will focus on different aspects such as the most popular types of properties, average rental prices, and busiest seasons for tourism. By gaining a deeper understanding of the Airbnb rental market in the Barossa Valley, we hope to provide valuable insights for property owners, tourists, and other stakeholders interested in the region. Additionally, identifying patterns and trends in the market will provide valuable information for those looking to invest in or enter the Barossa Valley property market on Airbnb.</span>

<span style="font-size: 10.5pt; color: black; font-family: &quot;Calibri(Body)&quot;, serif;"><b>Actions we will take in this project:</b></span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-Clean and organize the dataset using SQL</span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-Analyze the most popular types of properties available for rent</span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-Determine the average rental prices for properties in the Barossa Valley</span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-Determine the financial performance for Airbnb properties in the Barossa Valley<br></span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-Determine the financial performance for Airbnb hosts in the Barossa Valley<br></span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-Investigate patterns and trends in the Barossa Valley property market on Airbnb</span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-Provide valuable insights for property owners, tourists, and other stakeholders interested in the region.</span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-Identify opportunities for investment in the Barossa Valley property market on Airbnb.</span>

**<span style="font-size:10.5pt;font-family:&quot;Calibri body&quot;,serif;mso-fareast-font-family:
&quot;Times New Roman&quot;;mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-fareast-language:
PT-BR">Table of Contents</span>**

<span style="font-size: 10.5pt; font-family: &quot;Calibri body&quot;, serif; color: black;">1.<span style="font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: &quot;Times New Roman&quot;;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span> **<span style="font-size:10.5pt;font-family:
&quot;Calibri body&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;mso-bidi-font-family:
&quot;Segoe UI&quot;;color:black;mso-fareast-language:PT-BR">Data Quality Check and Cleaning</span>**

 <span lang="EN-AU" style="font-size:10.5pt;font-family:&quot;Calibri body&quot;,serif;mso-fareast-font-family:
&quot;Times New Roman&quot;;mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-ansi-language:
EN-AU;mso-fareast-language:PT-BR">&nbsp;Diagnosticing data issues | Dropping columns&nbsp; | Dropping missing values</span>

<span style="font-size: 10.5pt; font-family: &quot;Calibri body&quot;, serif; color: black;">2.<span style="font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: &quot;Times New Roman&quot;;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span> **<span style="font-size:10.5pt;font-family:
&quot;Calibri body&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;mso-bidi-font-family:
&quot;Segoe UI&quot;;color:black;mso-fareast-language:PT-BR">Descriptive Analysis</span>**

<span lang="EN-AU" style="font-size:10.5pt;font-family:&quot;Calibri body&quot;,serif;mso-fareast-font-family:
&quot;Times New Roman&quot;;mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-ansi-language:
EN-AU;mso-fareast-language:PT-BR">Summary statistic of price|&nbsp; Frequency table of rental price&nbsp;</span> 

<span style="font-size:10.5pt;font-family:&quot;Calibri body&quot;,serif;mso-fareast-font-family:
&quot;Calibri body&quot;;mso-bidi-font-family:&quot;Calibri body&quot;;color:black;mso-fareast-language:
PT-BR">3.<span style="font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: &quot;Times New Roman&quot;;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span> **<span style="font-size:10.5pt;font-family:
&quot;Calibri body&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;mso-bidi-font-family:
&quot;Segoe UI&quot;;color:black;mso-fareast-language:PT-BR">Exploratory Analysis</span>**

<span style="font-size: 10.5pt; font-family: &quot;Calibri body&quot;, serif; color: black;">The best locations to invest| The best types of properties</span>

<span style="color: rgb(0, 0, 0); font-family: &quot;Calibri body&quot;, serif;">|B</span><span style="font-size: 10.5pt; color: black; font-family: &quot;Calibri(Body)&quot;, serif;">ehavior of the best hosts in the market</span><span style="color: rgb(0, 0, 0); font-family: &quot;Calibri body&quot;, serif;">| Properties under $100</span>

<span style="font-size: 10.5pt; font-family: &quot;Calibri body&quot;, serif; color: black;">4.<span style="font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: &quot;Times New Roman&quot;;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span> **<span style="font-size:10.5pt;font-family:
&quot;Calibri body&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;mso-bidi-font-family:
&quot;Segoe UI&quot;;color:black;mso-fareast-language:PT-BR">Conclusion</span>**

<span style="font-size:10.5pt;font-family:&quot;Calibri body&quot;,serif;mso-fareast-font-family:
&quot;Calibri body&quot;;mso-bidi-font-family:&quot;Calibri body&quot;;color:black;mso-fareast-language:
PT-BR">5.<span style="font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: &quot;Times New Roman&quot;;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span> **<span style="font-size:10.5pt;font-family:
&quot;Calibri body&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;mso-bidi-font-family:
&quot;Segoe UI&quot;;color:black;mso-fareast-language:PT-BR">References</span>**

**DATA QUALITY CHECK AND CLEANING THE DATASET**

In [130]:
-- AN OVERVIEW OF BAROSSA VALLEY AIRBNB DATASET, THE FIRST 20 ROWS.
select TOP(20)*
FROM [listings (1)]

(20 rows affected)

Total execution time: 00:00:00.061

id name host_id host_name neighbourhood latitude longitude room_type price minimum_nights number_of_reviews last_review reviews_per_month calculated_host_listings_count availability_365 number_of_reviews_ltm 1125097 Strathlyn Coach House Barossa 6172214 Heather Angaston -3449389 13902942 Entire home/apt 254 2 166 2022-09-19 151 3 302 24 1133195 Strathlyn Lemon Tree Spa Barossa 6172214 Heather Penrice -3449403 13903202 Entire home/apt 236 2 59 2022-09-04 61 3 323 14 1133236 Strathlyn Spa Garden Suites Barossa 6172214 Heather Penrice -3449356 13903221 Private room 260 1 49 2022-09-18 47 3 318 15 1178384 Country cottage getaway 2122575 Tanya Nuriootpa -3446694 1389936 Private room 104 1 0 NULL NULL 1 335 0 1710462 Shea-Oak Log Studio Shed 6645956 Kylie And Tom Shea-Oak Log -3451049 13881731 Entire home/apt 102 2 497 2022-09-06 459 1 330 30 1787389 Barossa Valley Apt 1 offers 2 bedrooms 2 bathroom 9380091 Kym Tanunda -3452835 13895364 Entire home/apt 400 2 17 2020-12-04 21 17 50 0 2028951 2 bedrooms with Woodfire, Barossa Country Cottages 10421165 Mark And Fiona Lyndoch -3460374 13888348 Entire home/apt 209 2 8 2022-09-05 145 2 309 8 2327616 Gawler River Farm B and B. 6713275 Bill Gawler River -3460563 13864823 Entire home/apt 90 1 125 2022-09-21 135 1 343 29 2567051 The Miners Cottage Barossa Valley 10726293 Rebecca Cockatoo Valley -3463884 13884564 Private room 210 2 31 2022-08-22 32 1 267 5 3506622 A nice room in a big 4 b/room house 17656515 Andrew Buchfelde -3461874 13873199 Private room 85 1 0 NULL NULL 1 365 0 5289072 Mt McKenzie Raspberry Farm 27321203 Phillip Flaxman Valley -3456405 13907988 Entire home/apt 200 3 119 2022-08-24 141 1 134 20 5426792 "The Shed" 28149709 Andrew & Belinda Nuriootpa -3446397 13900191 Entire home/apt 63 1 138 2022-09-11 166 1 219 15 5537173 1837 Barossa Luxury Vineyard Cottage S Australia 28712482 Nelly Lyndoch -3458347 13886353 Entire home/apt 425 2 34 2022-09-17 70 3 303 11 6157653 Barossa Barn Bed & Breakfast 31945359 Julie Penrice -344877 13904407 Entire home/apt 277 2 28 2022-04-19 32 1 80 3 6170279 Little Para Cottage 32013001 Wendy Tanunda -3451062 13897196 Entire home/apt 206 2 160 2022-09-11 177 1 6 16 6170887 Elderton Wines Guest House 32017867 Elderton Nuriootpa -3447628 13899761 Entire home/apt 350 1 30 2022-08-08 34 1 214 4 6296823 Cosy 2-bedroom Cottage in the heart of the Barossa 32743215 Veritas Retreat Barossa Tanunda -3450314 13895909 Entire home/apt 290 2 80 2022-09-19 89 2 270 11 6861247 Rustic on 13th 35954891 Kirsten Buchfelde -3460501 13874353 Entire home/apt 239 4 46 2022-05-05 63 1 152 2 7055506 Vineyard Cottage - Under the Vines 10626104 Karen Nuriootpa -3450166 13899752 Entire home/apt 243 2 180 2021-10-17 207 2 0 1 7297171 Pembury Chase Barossa Valley 26147078 Jonathan And Kate Angaston -3450182 1390374 Private room 95 1 150 2019-04-28 174 6 0 0

In [131]:
--INSPECTING THE DUPLICATES ON THE PRYMARY KEY COLUMN "ID".
-- Verifying if there is only one unique id for each property in the list. 
SELECT COUNT(DISTINCT id)as 'Unique rows',
COUNT(*) as 'Total Rows'
FROM [listings (1)]

(1 row affected)

Total execution time: 00:00:00.009

Unique rows Total Rows 324 324

In [132]:
--CHECKING THE NULL VALUES ON THE DATASET.
--Percentage of null values on reviews_per_month, price, availability_365,neighbourhood.
SELECT ROUND((COUNT(CASE WHEN reviews_per_month IS NULL THEN 1 END) * 100.0) / COUNT(*),2) as reviews_per_month_null_percentage,
       (COUNT(CASE WHEN  price IS NULL THEN 1 END) * 100.0) / COUNT(*) as price_null_percentage,
       (COUNT(CASE WHEN availability_365 IS NULL THEN 1 END) * 100.0) / COUNT(*) as availability_365_null_percentage,
       (COUNT(CASE WHEN neighbourhood IS NULL THEN 1 END) * 100.0) / COUNT(*) as neighbourhood_null_percentage
FROM [listings (1)]



Warning: Null value is eliminated by an aggregate or other SET operation.

(1 row affected)

Total execution time: 00:00:00.019

reviews_per_month_null_percentage price_null_percentage availability_365_null_percentage neighbourhood_null_percentage 11.730000000000 0.000000000000 0.000000000000 0.000000000000

**Comments:** The "reviews per month" feature in our analysis has a null value rate of 11.73%, which is greater than the 5% acceptance rate we have set for our analysis. However, this does not significantly impact our overall analysis, and we will proceed with including it in our analysis. During the specific analysis of reviews, we will exclude properties with null values for this feature.

**2\. DESCRIPTIVE ANALYSIS - SUMMARY STATISTICS**

In order to gain a deeper understanding of the rental prices of properties in <span style="background-color: rgb(255, 255, 0);">Barossa&nbsp;</span>  Valley, we will analyze the summary statistics and the table of frequencies including the mean, median, and standard deviation. These measures will give us a better idea of the distribution of prices in the region. With this information, we will be able to identify patterns and make more informed decisions about the real estate market in <mark>Baro</mark>ssa Valley. Understanding the distribution of prices will help us to better understand the prices of properties in the region and make better investment decisions.

In [133]:
-- OVERVIEW OF THE DATASET.
SELECT *
FROM [listings (1)]

(324 rows affected)

Total execution time: 00:00:00.040

id name host_id host_name neighbourhood latitude longitude room_type price minimum_nights number_of_reviews last_review reviews_per_month calculated_host_listings_count availability_365 number_of_reviews_ltm 1125097 Strathlyn Coach House Barossa 6172214 Heather Angaston -3449389 13902942 Entire home/apt 254 2 166 2022-09-19 151 3 302 24 1133195 Strathlyn Lemon Tree Spa Barossa 6172214 Heather Penrice -3449403 13903202 Entire home/apt 236 2 59 2022-09-04 61 3 323 14 1133236 Strathlyn Spa Garden Suites Barossa 6172214 Heather Penrice -3449356 13903221 Private room 260 1 49 2022-09-18 47 3 318 15 1178384 Country cottage getaway 2122575 Tanya Nuriootpa -3446694 1389936 Private room 104 1 0 NULL NULL 1 335 0 1710462 Shea-Oak Log Studio Shed 6645956 Kylie And Tom Shea-Oak Log -3451049 13881731 Entire home/apt 102 2 497 2022-09-06 459 1 330 30 1787389 Barossa Valley Apt 1 offers 2 bedrooms 2 bathroom 9380091 Kym Tanunda -3452835 13895364 Entire home/apt 400 2 17 2020-12-04 21 17 50 0 2028951 2 bedrooms with Woodfire, Barossa Country Cottages 10421165 Mark And Fiona Lyndoch -3460374 13888348 Entire home/apt 209 2 8 2022-09-05 145 2 309 8 2327616 Gawler River Farm B and B. 6713275 Bill Gawler River -3460563 13864823 Entire home/apt 90 1 125 2022-09-21 135 1 343 29 2567051 The Miners Cottage Barossa Valley 10726293 Rebecca Cockatoo Valley -3463884 13884564 Private room 210 2 31 2022-08-22 32 1 267 5 3506622 A nice room in a big 4 b/room house 17656515 Andrew Buchfelde -3461874 13873199 Private room 85 1 0 NULL NULL 1 365 0 5289072 Mt McKenzie Raspberry Farm 27321203 Phillip Flaxman Valley -3456405 13907988 Entire home/apt 200 3 119 2022-08-24 141 1 134 20 5426792 "The Shed" 28149709 Andrew & Belinda Nuriootpa -3446397 13900191 Entire home/apt 63 1 138 2022-09-11 166 1 219 15 5537173 1837 Barossa Luxury Vineyard Cottage S Australia 28712482 Nelly Lyndoch -3458347 13886353 Entire home/apt 425 2 34 2022-09-17 70 3 303 11 6157653 Barossa Barn Bed & Breakfast 31945359 Julie Penrice -344877 13904407 Entire home/apt 277 2 28 2022-04-19 32 1 80 3 6170279 Little Para Cottage 32013001 Wendy Tanunda -3451062 13897196 Entire home/apt 206 2 160 2022-09-11 177 1 6 16 6170887 Elderton Wines Guest House 32017867 Elderton Nuriootpa -3447628 13899761 Entire home/apt 350 1 30 2022-08-08 34 1 214 4 6296823 Cosy 2-bedroom Cottage in the heart of the Barossa 32743215 Veritas Retreat Barossa Tanunda -3450314 13895909 Entire home/apt 290 2 80 2022-09-19 89 2 270 11 6861247 Rustic on 13th 35954891 Kirsten Buchfelde -3460501 13874353 Entire home/apt 239 4 46 2022-05-05 63 1 152 2 7055506 Vineyard Cottage - Under the Vines 10626104 Karen Nuriootpa -3450166 13899752 Entire home/apt 243 2 180 2021-10-17 207 2 0 1 7297171 Pembury Chase Barossa Valley 26147078 Jonathan And Kate Angaston -3450182 1390374 Private room 95 1 150 2019-04-28 174 6 0 0 8053447 Angels Place: Private Suite 42427624 Maryann Tanunda -3451719 13896402 Private room 165 1 82 2022-04-24 115 1 1 11 9264671 Barossa Idyll Studio -great for romantics 17055114 Tracey Tanunda -3452799 13895242 Private room 146 2 27 2021-04-22 34 4 173 0 9319789 Barossa Idyll -centre of town, breathtaking views 17055114 Tracey Tanunda -3452821 138953 Entire home/apt 757 1 80 2022-09-16 103 4 210 20 9319994 Barossa Idyll 3 beds - town centre, amazing views 17055114 Tracey Tanunda -3452935 13895426 Entire home/apt 413 2 54 2022-09-15 71 4 203 17 9453124 Barossa Idyll 4 beds - town centre, amazing views 17055114 Tracey Tanunda -3452906 13895436 Entire home/apt 651 1 29 2022-08-05 35 4 209 11 9714178 Barossa Bed & Breakfast 50149490 Yvette Tanunda -3452206 13896239 Entire home/apt 515 1 57 2022-09-10 73 1 217 25 9850994 Pembury Homestead - Barossa Valley- 3 bedrooms 26147078 Jonathan And Kate Angaston -3449908 13904063 Entire home/apt 120 1 7 2016-01-13 9 6 0 0 10285222 Barossa Dreams Ruby Suite full kitchen in Tanunda 9380091 Kym Tanunda -3452708 13895417 Entire home/apt 275 2 10 2020-03-18 12 17 4 0 10287444 Barossa Dreams 

**<u>\-Summary Statistics of the data properties rent price</u>**

In [134]:
--SUMMARY STATISTICS OF RENT PRICE.  
SELECT SUM(price) as sum_price,
AVG(price) as mean_price,
MAX(price) as max_price,
MIN(price) as min_price,
VAR(price) as variance_price,
STDEV(price) as standart_dev_price,
MAX(price)- MIN(price) as range_price
FROM [listings (1)]


(1 row affected)

Total execution time: 00:00:00.014

sum_price mean_price max_price min_price variance_price standart_dev_price range_price 111062 342 2800 63 91814,76432366318 303,0095119359509 2737

In [135]:
--INTERQUATILE RANGE AND MEDIAN OF RENT PRICE 
SELECT TOP(1) price,
PERCENTILE_CONT(0.25) WITHIN GROUP (ORDER BY price) OVER() AS first_quartile,
PERCENTILE_CONT(0.5) WITHIN GROUP (ORDER BY price) OVER() AS median_price,
PERCENTILE_CONT(0.75) WITHIN GROUP (ORDER BY price) OVER() AS third_quartile
FROM [listings (1)]
GROUP BY price;

(1 row affected)

Total execution time: 00:00:00.033

price first_quartile median_price third_quartile 63 190,75 271,5 424,25

**<u>\-Frequency table of the properties rent price</u>**

**Comments:** As a result, the distribution of the data is right-skewed, with the majority of the values concentrated on the lower end and a few outliers on the higher end. The mean is greater than the median. Additionally, the interquartile range (IQR) is $233.50, indicating that 50% of the data is concentrated between $190.75 and $424.25.

In [136]:
-- FREQUENCY TABLE OF THE PRICE RENT AIRBNB AT BAROSSA VALLEY.
WITH frequency_Table AS (
    SELECT
      CASE 
        WHEN price < 150 THEN '0-150'
        WHEN price BETWEEN 151 AND 300 THEN '151-300'
        WHEN price BETWEEN 301 AND 450 THEN '301-450'
        WHEN price BETWEEN 451 AND 600 THEN '451-600'
        WHEN price BETWEEN 601 AND 750 THEN '601-750'
        WHEN price BETWEEN 751 AND 900 THEN '751-900'
        WHEN price BETWEEN 901 AND 1050 THEN '901-1050'
        WHEN price BETWEEN 1051 AND 1200 THEN '1051-1200'
        WHEN price BETWEEN 1201 AND 1500 THEN '1201-1500'
        WHEN price > 1500 THEN '>= 1500'
        ELSE 'Other'
      END AS price_range,
        COUNT(*) as frequency
    FROM [listings (1)]
    GROUP BY price
)
-- IT WAS NOT POSSIBLE SORT THE CREATED RANGE COLUMN ABOVE (price_range) BECAUSE '0-150' IT IS STRING DATA TYPE. SO I CREATED OTHER COLUMN WITH THE CORRECT SORT ORDER.
SELECT price_range, 
SUM(frequency) as count_freq,
CASE 
    WHEN price_range = '0-150' THEN 1
    WHEN price_range = '151-300' THEN 2
    WHEN price_range = '301-450' THEN 3
    WHEN price_range = '451-600' THEN 4
    WHEN price_range = '601-750' THEN 5
    WHEN price_range = '751-900' THEN 6
    WHEN price_range = '901-1050' THEN 7
    WHEN price_range = '1051-1200' THEN 8
     WHEN price_range = '1201-1500' THEN 9
    WHEN price_range = '>= 1500' THEN 10
    ELSE 11
  END AS sort_order
FROM frequency_Table
GROUP BY price_range
ORDER BY sort_order;

(11 rows affected)

Total execution time: 00:00:00.020

price_range count_freq sort_order 0-150 50 1 151-300 150 2 301-450 59 3 451-600 23 4 601-750 10 5 751-900 13 6 901-1050 4 7 1051-1200 1 8 1201-1500 4 9 >= 1500 4 10 Other 6 11

**Comments:** In summary, the analysis of the price distribution of properties in <mark><mark><mark>Baro</mark></mark></mark>ssa Valley suggests a wide availability of fair-cost options for accommodation. The data shows that the majority of properties, 79.93%, have a price range that falls between $0 to $450 per night, with 46% of the properties having a price range between $150 to $300 per night. The average price of rent in <mark><mark><mark><mark><mark>Baro</mark></mark></mark></mark></mark>ssa Valley is $342 per night, which is considered as a fair price for accommodation. This indicates that <mark><mark><mark><mark><mark><mark><mark><mark><mark>Baro</mark></mark></mark></mark></mark></mark></mark></mark></mark>ssa Valley is a destination where visitors can find a variety of fair-cost options for accommodation and budget travelers can also find suitable options. Also, it is worth noting that while there are only a few properties that have a high cost, costing more than $1500 per night, they should not be considered as typical prices in <mark><mark><mark><mark><mark><mark>Ba<mark>ro</mark></mark></mark></mark></mark></mark></mark><mark>ssa</mark> Valley.

**3\. EXPLORATORY ANALYSIS**

**<u>\- Exploring the best regions for a investment in the Barossa Valley rental market on Airbnb.</u>**

**<u>\-Count of listings per neighbourhood</u>**

In [137]:
--INVESTIGATING THE QUANTITY OF LISTINGS PER NEIHGBOURHOOD
SELECT neighbourhood,
COUNT(id) as listings_count
FROM [listings (1)]
GROUP BY neighbourhood
ORDER BY listings_count DESC

(26 rows affected)

Total execution time: 00:00:00.010

neighbourhood listings_count Tanunda 89 Angaston 53 Lyndoch 21 Nuriootpa 17 Kapunda 15 Buchfelde 15 Marananga 14 Bethany 12 Concordia 11 Mount Crawford 8 Mount Pleasant 8 Springton 8 Williamstown 6 Hewett 6 Moculta 5 Light Pass 5 Penrice 5 Kalbeeba 4 Altona 4 Greenock 4 Eden Valley 3 Flaxman Valley 3 Sandy Creek 3 Gawler River 2 Cockatoo Valley 2 Shea-Oak Log 1

**<u>\-Average price per Neighbourhood</u>**

In [5]:
--INVESTIGATING THE AVERAGE PRICE AND NUMBER OF PROPERTIES LISTED PER NEIHGBOURHOOD 
SELECT neighbourhood,
AVG(price) as avg_price,
count(id) as count_listings
FROM [listings (1)]
GROUP BY neighbourhood
ORDER BY count_listings DESC

(26 rows affected)

Total execution time: 00:00:00.030

neighbourhood avg_price count_listings Tanunda 416 89 Angaston 313 53 Lyndoch 408 21 Nuriootpa 275 17 Kapunda 448 15 Buchfelde 148 15 Marananga 426 14 Bethany 296 12 Concordia 165 11 Mount Crawford 202 8 Mount Pleasant 472 8 Springton 216 8 Williamstown 250 6 Hewett 257 6 Moculta 320 5 Light Pass 391 5 Penrice 261 5 Kalbeeba 216 4 Altona 913 4 Greenock 245 4 Eden Valley 181 3 Flaxman Valley 217 3 Sandy Creek 240 3 Gawler River 110 2 Cockatoo Valley 502 2 Shea-Oak Log 102 1

**Comments**: The towns of Tanunda and Angaston are the heart of the Barossa Valley region, containing 43.82% of the properties listed in the area, making them an important destination for visitors and potential investors. They offer a wide range of qualitative attributes such as rich history, renowned wine production, delicious food, art, scenic views, festivals, lively community, and a range of accommodation options. Their history, heritage sites, wineries, food, art, and events contribute to the unique and attractive atmosphere of the region. As we can see It has a range of accommodation options available and the region is accessible to everyone. All these attributes make the Barossa Valley region, and particularly the towns of Tanunda and Angaston, a great place to visit, live and invest. Also the average price in Tanunda is 416 dollars and in Angaston is 313 dollars. We have also noticed that regions that are farther away from the central axis of Tanunda-Angaston-Lyndoch have fewer properties available for rent on Airbnb. The average price of those regions such as Cockatoo Valley and Altona, which only have 2 and 4 properties listed respectively, may not be accurate due to the limited quantity of properties. To make our analysis more accurate, we will only consider from now regions that have more than 5 properties listed to provide more accurate occupancy rate and properties under 1500 dollars (atypical price) in order to provide a more reliable average price to each neighbourhood.

**<u>\-Resonable average price per Neighbourhood(Exclude Outliers)</u>**

In [139]:
-- NUMBER OF PROPERTIES LISTED PER NEIHGBOURHOOD(> 5 PROPERTIES) AND PRICE (< $1500)
SELECT neighbourhood,
AVG(price) as avg_price,
count(id) as count_listings
FROM [listings (1)]
WHERE price < 1500
GROUP BY neighbourhood
HAVING count(id) > 5
ORDER BY count_listings DESC


(14 rows affected)

Total execution time: 00:00:00.010

neighbourhood avg_price count_listings Tanunda 359 86 Angaston 283 52 Lyndoch 408 21 Nuriootpa 275 17 Buchfelde 148 15 Marananga 426 14 Kapunda 373 14 Bethany 296 12 Concordia 165 11 Mount Crawford 202 8 Mount Pleasant 472 8 Springton 216 8 Williamstown 250 6 Hewett 257 6

**Comments:** The average value per neighbourhood can be affected by the number of listings in each neighbourhood as well as by outliers, such as properties with unusually high nightly rates. These high-priced properties do not accurately represent the average price and are not representative of the neighbourhood as a whole. To provide a more consistent average price per neighbourhood, we excluded properties with a nightly rate greater than $1500.

**<u>\-Occupation rate in each neighbourhood of Barossa Valley (Exclude outliers)</u>**

In [140]:
--AVERAGE OCCUPATION RATE PER NEIGHBOURHOOD(> 5 properties)
WITH occupancy AS (
    SELECT neighbourhood,
    (365-AVG(availability_365)) as occupated_days
    FROM [listings (1)]
    GROUP BY neighbourhood
    HAVING count(id) > 5
    )
SELECT neighbourhood,
ROUND((occupated_days/CAST(365 as float)*100),2) as occupancy_rate
FROM occupancy
ORDER BY occupancy_rate DESC

(14 rows affected)

Total execution time: 00:00:00.013

neighbourhood occupancy_rate Springton 74,79 Bethany 57,81 Nuriootpa 51,51 Tanunda 49,32 Buchfelde 37,53 Angaston 36,16 Concordia 35,34 Williamstown 35,34 Lyndoch 34,79 Marananga 32,33 Kapunda 28,49 Hewett 26,03 Mount Pleasant 21,92 Mount Crawford 21,1

In [142]:
-- ZOOM ON BETHANY AND SPRIGTON PROPERTIES
SELECT id,price,neighbourhood
FROM [listings (1)]
WHERE neighbourhood = 'Bethany' OR neighbourhood = 'Springton'

(20 rows affected)

Total execution time: 00:00:00.008

id price neighbourhood 14031240 330 Bethany 17737018 195 Springton 36464062 795 Bethany 40212692 250 Bethany 45548496 420 Springton 45714238 165 Springton 45714531 300 Springton 48061876 176 Springton 49448518 179 Springton 49913465 1029 Bethany 53055342 150 Springton 53245474 217 Bethany 53498238 150 Springton 702031267609659440 115 Bethany 703395754090693752 138 Bethany 703454945429923296 202 Bethany 703872734491160951 138 Bethany 703873092978064872 115 Bethany 703873217398549559 115 Bethany 703873326508107679 115 Bethany

**Comments:** The result of our analysis shows that the region of Bethany, Springton, and Nuriotpa in the Barossa Valley region of South Australia have high occupancy rates for short-term rentals on Airbnb. These regions are known for their attractions such as wineries, restaurants, and hotels. However, when compared to larger region like Tanunda and Angaston, the number of listings on Airbnb in these regions is relatively small. This means that the occupancy rate in these regions might appear higher than it actually is because of the fewer number of listings. Although, it's also worth noting that region like Tanunda and Angaston have a relatively medium occupancy rate which suggests that there is still demand for short-term rentals in these areas due the high count of properties available. It's important to consider the number of listings on Airbnb when evaluating the potential return on investment for a property in these areas.

**<u>\-Financial rent performance (gross revenue) of properties in each neighbourhood</u>**

In [141]:
-- AVERAGE GROSS REVENUE PER NEIGHBOURHOOD
WITH occupancy AS (
    SELECT neighbourhood,
    AVG(price) as avg_price,
    (365-AVG(availability_365)) as occupated_days,
    COUNT(id) AS count_listings
    FROM [listings (1)]
    WHERE price < 1500
    GROUP BY neighbourhood
    HAVING count(id) > 5
    )
    SELECT neighbourhood,
    avg_price,
    ROUND((count_listings)*(occupated_days)*(avg_price),2) as gross_revenue_year,
    (ROUND((count_listings)*(occupated_days)*(avg_price),2))/365 as gross_revenue_daily_365days
    FROM occupancy
    ORDER BY gross_revenue_year DESC
  
   
        
    

(14 rows affected)

Total execution time: 00:00:00.012

neighbourhood avg_price gross_revenue_year gross_revenue_daily_365days Tanunda 359 5372076 14718 Angaston 283 1957228 5362 Lyndoch 408 1088136 2981 Nuriootpa 275 878900 2407 Bethany 296 749472 2053 Marananga 426 703752 1928 Kapunda 373 558754 1530 Springton 216 471744 1292 Buchfelde 148 304140 833 Mount Pleasant 472 302080 827 Concordia 165 234135 641 Williamstown 250 193500 530 Hewett 257 146490 401 Mount Crawford 202 124432 340

**Comments:** The Barossa Valley region in South Australia presents a promising opportunity for property investment in the short-term rental market on Airbnb, with an annual revenue of over $700,000 for the top 5 regions and $5M forTanunda. The regions Bethany, Springton, and Nuriotpa have high occupancy rates. However, it's important to keep in mind that the number of listings in these regions is relatively small when compared to larger regions like Tanunda and Angaston. This means that the occupancy rate in these small regions might was impacted because of the fewer number of listings. For an accurate analysis and safe results, it's important to compare these regions to similar ones in the same region with a similar number of listings on Airbnb. Therefore, it's crucial to consider the number of listings, occupancy rate, and location when evaluating the potential revenue and return on investment for properties in these areas.Overall, a comprehensive analysis of all these factors is essential when evaluating the potential of a property for short-term rental in the Barossa Valley region.

**<u>\-The high prices properties and where are they located</u>**

In [1]:
-- HIGH PRICES LISTINGS ON AIRBNB AT BAROSSA VALLEY
SELECT id,
neighbourhood,
price
FROM [listings (1)]
WHERE price > 1200


(8 rows affected)

Total execution time: 00:00:00.043

id neighbourhood price 14969874 Tanunda 1414 20175507 Tanunda 1920 20176443 Tanunda 2800 25893345 Tanunda 1500 48999138 Kapunda 1500 52923843 Angaston 1886 634520727307763774 Altona 1603 668396960497325094 Lyndoch 1367

**Comments:** <span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">Our analysis found that half of the luxury properties in the region are located in Tanunda, consistent with the high number of visitors to the area. The high demand for luxury properties in a popular tourist destination like Tanunda is likely due to both the area's high visitor numbers and its desirable location.</span>

**<u>\-Analysis on the most interesting type of properties to investment at Barossa Valley</u>**

**<u>\-Types of properties on the dataset</u>**

In [143]:
-- TYPES OF ROOM AVAILABLE
SELECT DISTINCT(room_type)
FROM [listings (1)]

(4 rows affected)

Total execution time: 00:00:00.008

room_type Entire home/apt Hotel room Private room Shared room

**Comments:** There are 4 type of room available for properties on airbnb list.

In [144]:
--EXPLORING THE MOST COMMON TYPES OF PROPERTIES (ROOM_TYPE)
SELECT 
ROUND(COUNT(CASE WHEN room_type = 'Entire home/apt' THEN 1 END)/ CAST(COUNT(*) AS float),4)*100 AS "Entire home/apt(%)",
ROUND(COUNT(CASE WHEN room_type = 'Hotel room' THEN 1 END)/ CAST(COUNT(*) AS float),4)*100 AS "Hotel room(%)",
ROUND(COUNT(CASE WHEN room_type = 'Private room' THEN 1 END)/ CAST(COUNT(*) AS float),4)*100 AS "Private room(%)",
ROUND(COUNT(CASE WHEN room_type = 'Shared room' THEN 1 END)/ CAST(COUNT(*) AS float),4)*100 AS "Shared room(%)"
FROM [listings (1)];

Warning: Null value is eliminated by an aggregate or other SET operation.

(1 row affected)

Total execution time: 00:00:00.009

Entire home/apt(%) Hotel room(%) Private room(%) Shared room(%) 87,96000000000001 0,31 11,42 0,31

**Comments:** Between the types of rooms 87.96% is Entire Home/apt and 11.42% is Private room . Those two types of rooms are the most common at Barossa Valley.

**<u>\-The two main type of properties available in the dataset and where they are located</u>**

In [6]:
--NUMBER OF LISTINGS OF ENTIRE HOME/APT AND PRIVATE ROOM 
SELECT neighbourhood,
COUNT(*) as listings_count,
COUNT(CASE WHEN room_type = 'Entire home/apt' THEN 1 END) AS 'Entire home/apt_count',
COUNT(CASE WHEN room_type = 'Private room' THEN 1 END) AS 'Private room_count'
FROM [listings (1)]
WHERE price < 1500
GROUP BY neighbourhood
HAVING COUNT(*) > 5 
ORDER BY listings_count DESC


Warning: Null value is eliminated by an aggregate or other SET operation.

(14 rows affected)

Total execution time: 00:00:00.016

neighbourhood listings_count Entire home/apt_count Private room_count Tanunda 86 81 5 Angaston 52 41 11 Lyndoch 21 19 2 Nuriootpa 17 13 4 Buchfelde 15 11 4 Marananga 14 14 0 Kapunda 14 11 3 Bethany 12 11 0 Concordia 11 11 0 Mount Crawford 8 7 1 Mount Pleasant 8 3 4 Springton 8 8 0 Williamstown 6 6 0 Hewett 6 6 0

**Comments:** The region of Tanunda, Angaston, and Lyndoch is a prime location for rental properties. The data shows a high concentration of listings for both Entire homes/apartments, as well as Private rooms. The majority of these listings are for entire homes and apartments, indicating a strong demand for this type of accommodation in the area. This could be attributed to the area's proximity to popular attractions and overall quality of life. The diversity of housing options available in this region makes it an attractive location for individuals with different needs and preferences.

**<u>\-Financial rent performance Entire home/apt</u>**

In [146]:
-- ANNUAL_REVENUE, DAILY_REVENUE, DAILY_REVENUE_FOR_EACH_PROPERTY OF ENTIRE HOME/APT
SELECT neighbourhood,
COUNT(CASE WHEN room_type = 'Entire home/apt' THEN 1 END) AS 'Entire home/apt_count',
(COUNT(CASE WHEN room_type = 'Entire home/apt' THEN 1 END)*AVG(Price))*(365-AVG(availability_365)) AS 'Entire home/apt_avg_revenue',
(COUNT(CASE WHEN room_type = 'Entire home/apt' THEN 1 END)*AVG(Price))*(365-AVG(availability_365))/365 AS 'Entire home/apt_avg_revenue_daily_365days',
((COUNT(CASE WHEN room_type = 'Entire home/apt' THEN 1 END)*AVG(Price))*(365-AVG(availability_365))/365)/COUNT(CASE WHEN room_type = 'Entire home/apt' THEN 1 END) AS 'Entire home/apt_avg_revenue_daily_365days_eachproperty'
FROM [listings (1)]
WHERE price < 1500 AND room_type = 'Entire home/apt'
GROUP BY neighbourhood
HAVING COUNT(*) > 5 
ORDER BY [Entire home/apt_count] DESC


(13 rows affected)

Total execution time: 00:00:00.031

neighbourhood Entire home/apt_count Entire home/apt_avg_revenue Entire home/apt_avg_revenue_daily_365days Entire home/apt_avg_revenue_daily_365days_eachproperty Tanunda 81 5021352 13757 169 Angaston 41 1625650 4453 108 Lyndoch 19 955548 2617 137 Marananga 14 703752 1928 137 Nuriootpa 13 851838 2333 179 Bethany 11 660715 1810 164 Buchfelde 11 310189 849 77 Concordia 11 234135 641 58 Kapunda 11 476036 1304 118 Springton 8 471744 1292 161 Mount Crawford 7 102256 280 40 Hewett 6 146490 401 66 Williamstown 6 193500 530 88

**Comments:** T<span lang="EN-AU" style="font-size:11.0pt;line-height:
107%;font-family:&quot;Calibri&quot;,sans-serif;mso-ascii-theme-font:minor-latin;
mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-hansi-theme-font:
minor-latin;mso-bidi-font-family:&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;
mso-ansi-language:EN-AU;mso-fareast-language:EN-US;mso-bidi-language:AR-SA">he data indicates that, on average, in&nbsp;<span style="font-size: 14.6667px;">the region of Tanunda, Angaston, and Lyndoch</span>, entire home/apt rentals represent an impressive 90.31% of the gross revenue generated from properties rented on Airbnb. Therefore, it is clear that this type of property is a crucial aspect of the rental market in this region. The revenue data also supports this, showing substantial revenue from entire home and apartment rentals in each neighborhood. The table provides a clear picture of the gross revenue generated by these properties in each neighborhood, not considering the general costs, tax and mortgate of property .</span>

**<u>\-Financial rent performance(gross revenue) private room</u>**

In [147]:
-- ANNUAL_REVENUE, DAILY_REVENUE, DAILY_REVENUE_FOR_EACH_PROPERTY OF PRIVATE ROOM
SELECT neighbourhood,
COUNT(CASE WHEN room_type = 'Private Room' THEN 1 END) AS 'Private Room',
(COUNT(CASE WHEN room_type = 'Private Room' THEN 1 END)*AVG(Price))*(365-AVG(availability_365)) AS 'Private_Room_avg_revenue',
(COUNT(CASE WHEN room_type = 'Private Room' THEN 1 END)*AVG(Price))*(365-AVG(availability_365))/365 AS 'Private_Room_avg_revenue_daily_365days',
((COUNT(CASE WHEN room_type = 'Private Room' THEN 1 END)*AVG(Price))*(365-AVG(availability_365))/365)/COUNT(CASE WHEN room_type = 'Private Room' THEN 1 END) AS 'Private Room_avg_revenue_daily_365days_eachproperty'
FROM [listings (1)]
WHERE price < 1500 AND room_type = 'Private Room'
GROUP BY neighbourhood 


(11 rows affected)

Total execution time: 00:00:00.027

neighbourhood Private Room Private_Room_avg_revenue Private_Room_avg_revenue_daily_365days Private Room_avg_revenue_daily_365days_eachproperty Angaston 11 243936 668 60 Buchfelde 4 21976 60 15 Cockatoo Valley 1 20580 56 56 Kapunda 3 59856 163 54 Lyndoch 2 130000 356 178 Mount Crawford 1 12870 35 35 Mount Pleasant 4 0 0 0 Nuriootpa 4 63656 174 43 Penrice 1 12220 33 33 Sandy Creek 1 54626 149 149 Tanunda 5 248825 681 136

**Comments:** Our analysis shows that in the region of Tanunda, Angaston, and Lyndoch, private room rentals play an important role in the rental market, even though the gross revenue generated from this type of property may be smaller than that of Entire home/apartments. The data indicates that the quantity of properties of this type is not expressive in Tanunda, and it is more common in Angaston.

In [148]:
--CHECKING WHAT HAPPENED IN MOUNT PLEASANT
select neighbourhood,
room_type,
price,
availability_365 
from [listings (1)]
Where neighbourhood = 'Mount Pleasant' 

(8 rows affected)

Total execution time: 00:00:00.013

neighbourhood room_type price availability_365 Mount Pleasant Entire home/apt 315 91 Mount Pleasant Entire home/apt 105 0 Mount Pleasant Entire home/apt 1105 365 Mount Pleasant Private room 235 365 Mount Pleasant Shared room 288 365 Mount Pleasant Private room 432 365 Mount Pleasant Private room 720 365 Mount Pleasant Private room 576 365

**Comments:** The lack of occupancy in the four private rooms in Mount Pleasant throughout the year indicates a lower demand for this type of property.

Commands completed successfully.

Total execution time: 00:00:00

**<u>\-Investigating who are the currently hosts and how and where they invest on the rental market</u>**

In [150]:
-- OVERVIEW OF THE DATASET 
SELECT TOP(20)*
FROM [listings (1)]

(20 rows affected)

Total execution time: 00:00:00.009

id name host_id host_name neighbourhood latitude longitude room_type price minimum_nights number_of_reviews last_review reviews_per_month calculated_host_listings_count availability_365 number_of_reviews_ltm 1125097 Strathlyn Coach House Barossa 6172214 Heather Angaston -3449389 13902942 Entire home/apt 254 2 166 2022-09-19 151 3 302 24 1133195 Strathlyn Lemon Tree Spa Barossa 6172214 Heather Penrice -3449403 13903202 Entire home/apt 236 2 59 2022-09-04 61 3 323 14 1133236 Strathlyn Spa Garden Suites Barossa 6172214 Heather Penrice -3449356 13903221 Private room 260 1 49 2022-09-18 47 3 318 15 1178384 Country cottage getaway 2122575 Tanya Nuriootpa -3446694 1389936 Private room 104 1 0 NULL NULL 1 335 0 1710462 Shea-Oak Log Studio Shed 6645956 Kylie And Tom Shea-Oak Log -3451049 13881731 Entire home/apt 102 2 497 2022-09-06 459 1 330 30 1787389 Barossa Valley Apt 1 offers 2 bedrooms 2 bathroom 9380091 Kym Tanunda -3452835 13895364 Entire home/apt 400 2 17 2020-12-04 21 17 50 0 2028951 2 bedrooms with Woodfire, Barossa Country Cottages 10421165 Mark And Fiona Lyndoch -3460374 13888348 Entire home/apt 209 2 8 2022-09-05 145 2 309 8 2327616 Gawler River Farm B and B. 6713275 Bill Gawler River -3460563 13864823 Entire home/apt 90 1 125 2022-09-21 135 1 343 29 2567051 The Miners Cottage Barossa Valley 10726293 Rebecca Cockatoo Valley -3463884 13884564 Private room 210 2 31 2022-08-22 32 1 267 5 3506622 A nice room in a big 4 b/room house 17656515 Andrew Buchfelde -3461874 13873199 Private room 85 1 0 NULL NULL 1 365 0 5289072 Mt McKenzie Raspberry Farm 27321203 Phillip Flaxman Valley -3456405 13907988 Entire home/apt 200 3 119 2022-08-24 141 1 134 20 5426792 "The Shed" 28149709 Andrew & Belinda Nuriootpa -3446397 13900191 Entire home/apt 63 1 138 2022-09-11 166 1 219 15 5537173 1837 Barossa Luxury Vineyard Cottage S Australia 28712482 Nelly Lyndoch -3458347 13886353 Entire home/apt 425 2 34 2022-09-17 70 3 303 11 6157653 Barossa Barn Bed & Breakfast 31945359 Julie Penrice -344877 13904407 Entire home/apt 277 2 28 2022-04-19 32 1 80 3 6170279 Little Para Cottage 32013001 Wendy Tanunda -3451062 13897196 Entire home/apt 206 2 160 2022-09-11 177 1 6 16 6170887 Elderton Wines Guest House 32017867 Elderton Nuriootpa -3447628 13899761 Entire home/apt 350 1 30 2022-08-08 34 1 214 4 6296823 Cosy 2-bedroom Cottage in the heart of the Barossa 32743215 Veritas Retreat Barossa Tanunda -3450314 13895909 Entire home/apt 290 2 80 2022-09-19 89 2 270 11 6861247 Rustic on 13th 35954891 Kirsten Buchfelde -3460501 13874353 Entire home/apt 239 4 46 2022-05-05 63 1 152 2 7055506 Vineyard Cottage - Under the Vines 10626104 Karen Nuriootpa -3450166 13899752 Entire home/apt 243 2 180 2021-10-17 207 2 0 1 7297171 Pembury Chase Barossa Valley 26147078 Jonathan And Kate Angaston -3450182 1390374 Private room 95 1 150 2019-04-28 174 6 0 0

**<u>\-Selecting the 10 best hosts of the Airbnb list</u>**

In [151]:
--TOP 10 HOSTS WITH MORE PROPERTIES AT BAROSSA VALLEY
SELECT TOP(10)host_id, 
host_name,
SUM(calculated_host_listings_count)/COUNT(host_id) as num_of_listings
FROM [listings (1)]
GROUP BY host_id,host_name
ORDER BY num_of_listings DESC



(10 rows affected)

Total execution time: 00:00:00.014

host_id host_name num_of_listings 9380091 Kym 17 439331891 Emma&Roger 8 383527612 Jean-Claude 7 461601857 The Wine 7 38878329 Andrew 6 162433043 Danielle 6 472161981 Heather 6 26147078 Jonathan And Kate 6 63693332 GaryLeonie 5 262121622 Jerry 5

**<u>\-Where the 10 best hosts are located</u>**

In [152]:
WITH top10hosts as ( SELECT TOP(10)host_id, 
host_name,
SUM(calculated_host_listings_count)/COUNT(host_id) as num_of_listings
FROM [listings (1)]
GROUP BY host_id,host_name
ORDER BY num_of_listings DESC
) 
SELECT top10hosts.host_name, 
neighbourhood
FROM [listings (1)]
JOIN top10hosts
ON [listings (1)].host_id = top10hosts.host_id



(73 rows affected)

Total execution time: 00:00:00.018

host_name neighbourhood Kym Tanunda Jonathan And Kate Angaston Jonathan And Kate Angaston Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Kym Tanunda Jonathan And Kate Angaston Kym Tanunda Kym Tanunda GaryLeonie Buchfelde Andrew Concordia Jonathan And Kate Angaston Jonathan And Kate Lyndoch Jerry Tanunda Jerry Angaston Jerry Angaston Jerry Angaston Jerry Angaston Andrew Hewett Danielle Angaston Danielle Angaston Danielle Angaston Danielle Angaston Danielle Angaston Danielle Angaston Jonathan And Kate Angaston GaryLeonie Hewett Andrew Concordia Andrew Concordia GaryLeonie Kalbeeba Andrew Concordia GaryLeonie Hewett Jean-Claude Kapunda Jean-Claude Kapunda Jean-Claude Kapunda GaryLeonie Hewett Jean-Claude Kapunda Jean-Claude Kapunda Jean-Claude Kapunda Andrew Concordia Emma&Roger Nuriootpa Kym Tanunda Jean-Claude Kapunda The Wine Tanunda The Wine Tanunda The Wine Tanunda The Wine Tanunda The Wine Tanunda The Wine Tanunda The Wine Tanunda Heather Mount Pleasant Emma&Roger Bethany Emma&Roger Bethany Emma&Roger Bethany Emma&Roger Bethany Emma&Roger Bethany Emma&Roger Bethany Emma&Roger Bethany Heather Mount Pleasant Heather Mount Pleasant Heather Mount Pleasant Heather Mount Pleasant Heather Mount Pleasant

In [153]:
WITH top10hosts as (
    SELECT TOP(10)host_id, 
    host_name,
    SUM(calculated_host_listings_count)/COUNT(host_id) as num_of_listings
    FROM [listings (1)]
    GROUP BY host_id,host_name
    ORDER BY num_of_listings DESC
), 
neighbourtop10 AS (
    SELECT top10hosts.host_name, 
    neighbourhood
    FROM [listings (1)]
    JOIN top10hosts
    ON [listings (1)].host_id = top10hosts.host_id
) 
    SELECT ROUND((COUNT(CASE WHEN neighbourhood = 'Tanunda' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Tanunda(%)',
    ROUND((COUNT(CASE WHEN neighbourhood = 'Angaston' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Angaston(%)',
    ROUND((COUNT(CASE WHEN neighbourhood = 'Buchfelde' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Buchfelde(%)',
    ROUND((COUNT(CASE WHEN neighbourhood = 'Lyndoch' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Lyndoch(%)',
    ROUND((COUNT(CASE WHEN neighbourhood = 'Concordia' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Concordia(%)',
    ROUND((COUNT(CASE WHEN neighbourhood = 'Mount Pleasant' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Mount Pleasant(%)',
    ROUND((COUNT(CASE WHEN neighbourhood = 'Nuriootpa' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Nuriootpa(%)',
    ROUND((COUNT(CASE WHEN neighbourhood = 'Bethany' THEN 1 END) * 100.0) / CAST(COUNT(*) AS float),2) AS 'Bethany(%)'
FROM neighbourtop10

Warning: Null value is eliminated by an aggregate or other SET operation.

(1 row affected)

Total execution time: 00:00:00.017

Tanunda(%) Angaston(%) Buchfelde(%) Lyndoch(%) Concordia(%) Mount Pleasant(%) Nuriootpa(%) Bethany(%) 34,25 20,55 1,37 1,37 6,85 8,22 1,37 9,59

**Comments:** To gain a more comprehensive understanding of the property market in the Barossa Valley on Airbnb, we selected the top 10 hosts. By analyzing the number of listings and location of these hosts, we aim to identify patterns and trends in the market. Additionally, by examining the revenue generated by these hosts, we hope to gain insight into the financial performance of the market and how it may be impacted by various factors. From our analysis, it appears that Tanunda and Angaston are the preferred locations in the region for the main hosts on the list. This information can be useful for those looking to invest in or enter the Barossa Valley property market on Airbnb, as well as for current hosts looking to improve their performance.

**<u>\-Financial rent performance of the 10 best hosts(gross revenue)</u>**

In [154]:
-- TOP 10 FINANCIAL PERFORMANCE
SELECT TOP(10)
host_name,
COUNT(host_id) as num_of_listings,
AVG(price) as avg_price,
COUNT(host_name)*AVG(price)*(365-AVG(availability_365)) as annual_revenue_host,
(COUNT(host_name)*AVG(price)*(365-AVG(availability_365)))/ 365.0 as daily_revenue_host_365days
FROM [listings (1)]
GROUP BY host_name
ORDER BY num_of_listings DESC

(10 rows affected)

Total execution time: 00:00:00.014

host_name num_of_listings avg_price annual_revenue_host daily_revenue_host_365days Kym 17 749 3985429 10918.983561 Heather 11 414 241362 661.265753 Emma&Roger 8 168 318528 872.679452 Andrew 7 221 35581 97.482191 Jean-Claude 7 538 154406 423.030136 The Wine 7 250 56000 153.424657 Danielle 6 129 28638 78.460273 Jonathan And Kate 6 198 262548 719.309589 Karen 6 231 234234 641.736986 GaryLeonie 5 221 139230 381.452054

**Comments:** In this analysis, we have taken into account all properties listed by the top 10 hosts on Airbnb in the Barossa Valley, including outliers that have a higher number of listings than 1500. This approach allows us to provide more specific information for each host and better understand their individual performance. However, when we evaluate the revenue generated by the entire neighborhood, we exclude these outliers as they do not accurately represent the performance of the neighborhood as a whole.It appears that Kym, one of the top 10 hosts on Airbnb in the Barossa Valley, has significantly higher revenue numbers compared to the others. This suggests that there may be something unique or different about this host's properties or business strategy.

**<u>\-Checking the type,prices and where the properties of the best host 'Kym' is located.</u>**

In [155]:
--HOST "KYM" TOP (1)
SELECT id, neighbourhood, 
room_type,
host_name,
price,
availability_365
FRom [listings (1)]
WHERE host_name = 'Kym'

(17 rows affected)

Total execution time: 00:00:00.014

id neighbourhood room_type host_name price availability_365 1787389 Tanunda Entire home/apt Kym 400 50 10285222 Tanunda Entire home/apt Kym 275 4 10287444 Tanunda Entire home/apt Kym 350 3 10298152 Tanunda Entire home/apt Kym 350 12 10348508 Tanunda Entire home/apt Kym 350 16 10619599 Tanunda Entire home/apt Kym 275 4 10634089 Tanunda Private room Kym 275 4 11428035 Tanunda Entire home/apt Kym 500 269 14827811 Tanunda Entire home/apt Kym 400 54 14842846 Tanunda Entire home/apt Kym 379 58 14843749 Tanunda Entire home/apt Kym 400 53 14969874 Tanunda Entire home/apt Kym 1414 42 20175507 Tanunda Entire home/apt Kym 1920 28 20176443 Tanunda Entire home/apt Kym 2800 0 24813904 Tanunda Entire home/apt Kym 850 294 25893345 Tanunda Entire home/apt Kym 1500 1 542934522374568605 Tanunda Entire home/apt Kym 300 6

**Comments:**  By taking a closer look at Kym's properties, we can understand the reason behind their high revenue. Kym has listed 3 of the 4 most expensive night-rent properties in the Barossa Valley, with rates of $1500, $1920 and $2800 respectively. This suggests that their high revenue is likely due to their luxury properties and premium pricing strategy.

**<u>\- Examining the properties that have a nightly rental rate less than $100.</u>**

In [156]:
-- Properties under 100 dolars
SELECT neighbourhood,
room_type,
host_name,
price, 
availability_365
FROM [listings (1)]
WHERE price <= 100

(10 rows affected)

Total execution time: 00:00:00.014

neighbourhood room_type host_name price availability_365 Gawler River Entire home/apt Bill 90 343 Buchfelde Private room Andrew 85 365 Nuriootpa Entire home/apt Andrew & Belinda 63 219 Angaston Private room Jonathan And Kate 95 0 Angaston Private room Anthea 80 0 Buchfelde Private room Mary 90 129 Mount Crawford Entire home/apt Alan 100 69 Buchfelde Private room Ted 80 356 Nuriootpa Private room Jane 92 327 Buchfelde Private room Michelle 75 344

In [157]:
SELECT neighbourhood,
Count(id) as count_listing
FROM [listings (1)]
WHERE price < = 100
Group BY neighbourhood
Order by count_listing DESC

(5 rows affected)

Total execution time: 00:00:00.008

neighbourhood count_listing Buchfelde 4 Angaston 2 Nuriootpa 2 Gawler River 1 Mount Crawford 1

**Comments:** There are few options of properties under 100 dolars available.Buchfelde is on the outskirts of Adelaide , heading towards the center of Barossa Valley.This is one option for solo backpakers and adventurers.

**4\. CONCLUSION**

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-The dataset provides an overview of the Airbnb rental market in the Barossa Valley region. It allows us to analyze listings, prices, location, and occupancy rate, which can give us an understanding of the overall situation in the market. By combining this information with qualitative points such as wineries, restaurants, and visitor turnover, we can gain insights into the safest places to invest and the types of properties that are most commonly rented. It is important to note that the cost of properties is not included in the analysis as this information is not provided in the dataset. Therefore, to make informed investment decisions, it is necessary to consider the intersection of these characteristics with the financial gross rent performance of the regions.</span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">-</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">The analysis of the Barossa Valley region in South Australia shows that the mean price of night rental properties on Airbnb is $342, with a minimum price of $63 and a maximum price of $2800.</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">The interquartile range (IQR) is $233.50, indicating that 50% of the listings are between $190.75 and $424.25, this range represents a reasonable range of prices that one visitor could pay per night for a property in the Barossa Valley.</span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-T</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">here are a few high-priced properties with a maximum price of $2800. These high-priced properties represent a small percentage of the total listings and are mostly located in the town of Tanunda. This suggests that Tanunda might be known for its luxury options and amenities for visitors and tourists, which draws a specific type of visitors willing to pay higher prices for more upscale accommodations. This also indicates that Tanunda might have a higher number of high-end properties, and these luxury options might not be as prevalent in other towns of the region</span>

 <span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">&nbsp;-</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">Tanunda is a particularly attractive location for property investment in the short-term rental market on Airbnb. The town of Tanunda offers a wide range of qualitative attributes such as rich history, renowned wine production, delicious food, art, scenic views, festivals, lively community, and a range of accommodation options. These features contribute to the unique and attractive atmosphere of the region, making it a great place to visit, live, and invest.Furthermore, even with moderate occupancy rate, the properties in Tanunda generate an outstanding annual gross revenue of $5M, which is significantly higher compared to other regions in the Barossa Valley. This is due to the large quantity of listings in the region and the presence of luxury options and amenities, making Tanunda a safe place to invest in a property rental on Airbnb, considering all of these attributes.&nbsp;</span> <span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;"><br></span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">The analysis shows that the towns of Tanunda, Angaston, and Lyndoch are the principal locations in the heart of the Barossa Valley region. Angaston and Lyndoch have shown to be consistent options for investment as rental properties on Airbnb, despite the low occupancy rate in those regions. This is due to their high gross revenue of $1,957,228 and $1,088,136 respectively, which is a result of a good number of listings in those regions. This suggests that even with moderate occupancy rate, these regions are still able to generate a significant amount of revenue, making them a reliable option for investment.</span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">Nuriotpa is well located in the Barossa Valley, with fewer listings on Airbnb and lower average prices compared to other regions. However, it still offers potential for revenue generation and is a good option for budget-conscious travelers looking for affordable accommodation.</span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">Bethany and Springton have some of the highest occupancy rates in the Barossa Valley, due to their attractiveness as regions. Bethany is well-known for its wineries and is located in the center of the Barossa Valley, while Springton is located in the merge of the Eden Valley road. However, due to their small size, the number of listings available on Airbnb is limited, which might be reflected in the occupancy rate. Despite the small number of listings, Bethany still has a gross gross revenue of $749,472. Therefore, it's important to note that the occupancy rate in these regions cannot be directly compared to the occupancy rate of other regions with a larger number of listings, but it's still important to consider the revenue generated by the region.</span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-T</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">he most common type of properties available for rent on Airbnb in the Barossa Valley region are Entire Home/Apt, representing 87.96% of the properties listed. While Private Rooms also play a significant role in the rental market, it is clear that the Entire Home/Apt is the more popular option for visitors and investors in the region, as it offers a wide variety of prices and locations.</span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">-T</span><span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">he top ten hosts with the most listings on Airbnb in the Barossa Valley region, have a high proportion of their properties located in the towns of Tanunda, Angaston and Bethany respectively, with 34.25%, 20.55% and 9.59%. This suggests that these hosts have a significant presence in these regions and could indicate a competitive market in those specific areas.</span>

\-T<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">he host "Kym" has a significant presence on Airbnb in the Barossa Valley region, with 17 listings, which is among the highest number of listings in the region. Kym's properties have generated an outstanding annual gross revenue of $3M, which is a result of his strategy to invest in luxury properties in the region. Furthermore, Kym also owns 3 of the most expensive properties on the listings, located in Tanunda.</span>

**5\. REFERENCES**

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">Inside Airbnb (n.d.). Inside Airbnb: Data [online]. Inside Airbnb. Available at: http://insideairbnb.com/get-the-data/ [15/01/2023].<br></span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR">South Australia (n.d.). Destinations: Barossa [online]. South Australia. Available at: https://southaustralia.com/destinations/barossa [22/01/2023].</span>

<span style="color: black; font-family: &quot;Calibri(Body)&quot;, serif; font-size: 10.5pt;">Australia.com (n.d.). Guide to the Barossa Valley [online]. Australia. Available at: https://www.australia.com/en/places/adelaide-and-surrounds/guide-to-the-barossa-valley.html [23/01/2023].</span>

<span lang="EN-AU" style="font-size:10.5pt;
font-family:&quot;Calibri(Body)&quot;,serif;mso-fareast-font-family:&quot;Times New Roman&quot;;
mso-bidi-font-family:&quot;Segoe UI&quot;;color:black;mso-color-alt:windowtext;
mso-ansi-language:EN-AU;mso-fareast-language:PT-BR"><br></span>